include/fmt/color.h

```diff
@@ -492,6 +492,14 @@ template <typename Char> inline void reset_color(buffer<Char>& buffer) {
   buffer.append(reset_color.begin(), reset_color.end());
 }
 
+template <typename Arg> struct styled_arg {
+  FMT_CONSTEXPR styled_arg(const Arg& format_argument, text_style format_style)
+      : argument(format_argument), style(format_style) {}
```

Comment:
I don't think the constructor is needed because you could use aggregate initialization `styled_arg{...}` instead.

Reply:
You're right. I'm changing this right now